modules/admin-ui/src/main/webapp/styles/main.scss

```diff
@@ -19,7 +19,8 @@
  *
  */
 
-// bower:scss
+// bower:css
+@import "../../../../bower_components/angular-hotkeys/build/hotkeys.css";
```

Comment:
Hmm... shouldn't this be included from index.html?

Reply:
I will have to think about this again; maybe I did something wrong. But for me, that stylesheet was not included in the production build, while it worked fine in "proxy mode".

I will get back to you. :wink: